src/lib/twilio.js

```diff
@@ -0,0 +1,29 @@
+const sendTwilio = async (message) =>
+{
+  var request = require('request');
+  const settings = require("../config");  
+  var dataString = 'From='+settings.twilioData.fromPhoneNumber+'&To='+settings.twilioData.toPhoneNumber+'&Body='+message;
```

Comment:
This one is a complex string... You could be using a template literal. So your code becomes:

```
const dataString = `From=${settings.twilioData.fromPhoneNumber}&To=${settings.twilioData.toPhoneNumber}&Body=${message}`;
```